### DISCUSSION TITLE
How to make use of gdUnit4Mono API?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [MikeSchulze/gdUnit4Mono#29] [General]
Is there a need to provide brief documentation on how to make use of gdUnit4Mono API. 
Is the following a good example?
https://github.com/Delsin-Yu/GDTask.Nuget

**Comment by MikeSchulze** [comment]
The README.md is actually not updated and will be later include all necessary information.

**Reply by mushwrooom** [reply]
Hello. Is the API README.md getting an update anytime soon?

**Reply by GeorgeS2019** [reply]
Join discord...U will get faster response

**Comment by MikeSchulze** [comment]
> Hello. Is the API README.md getting an update anytime soon?

Check the main page ;)
https://github.com/MikeSchulze/gdUnit4Net/blob/master/README.md

**Reply by mushwrooom** [reply]
On main README, I follow the instruction on how to install the API:

> Checkout the [readme](https://github.com/MikeSchulze/gdUnit4Net/blob/master/api/README.md) to install the gdunit4.api package.

But the directed README doesn't contain any instruction on installation of the API, hence me asking.

**Comment by MikeSchulze** [comment]
The repository README is just the main entrance to link to the API and test adapter. ;)
You can also read it on the official [GdUnit4 documentation page](https://mikeschulze.github.io/gdUnit4/faq/vstest-adapter/).
